tools/tck-api/src/test/scala/org/opencypher/tools/tck/TckTest.scala

```diff
@@ -63,12 +63,17 @@ class TckTest extends AnyFunSpec with Assertions {
           CypherValueRecords.empty
         case ExecQuery if query.contains("foo()") =>
           ExecutionFailed(SYNTAX_ERROR, COMPILE_TIME, UNKNOWN_FUNCTION)
+        // assert that csv path parameter is not overwritten by additional parameters
+        case ExecQuery if query.contains("LOAD CSV") && params.keySet.equals(Set("param", "list")) =>
+          StringRecords(List("res"), List(Map("res" -> "[]"), Map("res" -> "[]")))
         case ExecQuery =>
           StringRecords(List("1"), List(Map("1" -> "1")))
       }
     }
 
     override def registerProcedure(signature: String, values: CypherValueRecords): Unit =
       ()
+
+    override def createCSVFile(contents: CypherValueRecords): String = "dummy/path.csv"
```

Comment:
I would test the whole roundtrip (multiple comments):

```suggestion
    override def createCSVFile(contents: CypherValueRecords): String = {
      cvsData = contents
      "dummy/path.csv"
    }
```